cmd/blockchaincmd/create.go

```diff
@@ -358,6 +360,22 @@ func createBlockchainConfig(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
+	// subnet-evm check based on genesis
+	// covers both subnet-evm vms and custom vms
+	if hasSubnetEVMGenesis, rawErr, err := app.HasSubnetEVMGenesis(blockchainName); rawErr != nil {
```

Comment:
Looking at `HasSubnetEVMGenesis`, it looks like the `rawErr` is the last return argument. Are these mixed up here?